kotest-common/src/jvmMain/kotlin/io/kotest/mpp/reflection.kt

```diff
@@ -23,6 +24,13 @@ object JvmReflection : Reflection {
 
    override fun paramNames(fn: Function<*>): List<String>? = fn.reflect()?.parameters?.mapNotNull { it.name }
 
+   override fun <T : Any> primaryConstructorMembers(klass: KClass<T>): List<Property> {
+      val dataClassConstructorParams = klass::constructors.get().last().parameters
```

Comment:
I think there is an actual method called primaryConstructor. Getting the last one might not be sufficnet.

Reply:
You're right, not sure how I missed that 🤦  Yeah I was worried about the edgecases with getting the last one.